src/app/pages/storage/volumes/volumes-list/volumes-list.component.ts

```diff
@@ -1153,7 +1153,8 @@ export class VolumesListTableConfig implements InputTableConf {
           onClick: (row) => {
             // open encryption options dialog
             const can_inherit = (row.parent && row.parent.encrypted);
-            const is_key = !row.is_passphrase;
+            const passphrase_parent = (row.parent && row.parent.key_format.value === 'PASSPHRASE');
```

Comment:
On a top-level dataset (/mnt/dataset) I get a console error and the dialog doesn't open because the parent doesn't have a key_format

![image](https://user-images.githubusercontent.com/9504493/79570101-d40fce00-8086-11ea-8292-64bcd6db7bac.png)